src/Illuminate/Support/Pluralizer.php

```diff
@@ -69,7 +69,7 @@ class Pluralizer
      */
     public static function plural($value, $count = 2)
     {
-        if ((int) abs($count) === 1 || static::uncountable($value)) {
+        if ((int) abs($count) === 1 || static::uncountable($value) || preg_match('/^(.*)[A-Za-z0-9\x{0080}-\x{FFFF}]$/u', $value) == 0) {
```

Comment:
Was this thoroughly reviewed?
- Initial anchor and pattern `^(.*)` is superfluous
- Simpler pattern is `/\p{Letter}$/u` is more readable
- The check could be done in `uncountable()`